Library/Homebrew/utils/git.rb

```diff
@@ -4,7 +4,13 @@ module Git
   module_function
 
   def last_revision_commit_of_file(repo, file, before_commit: nil)
-    args = [before_commit.nil? ? "--skip=1" : before_commit.split("..").first]
+    args = [
+      if before_commit.nil?
```

Comment:
Sorry to be a pain: can the `if` live outside the array; this it not typical Ruby usage. Instead consider something like
```ruby
args = if before_commit.nil?
  ["skip=1"]
```

Reply:
No worries. It's definitely better style.